tests/tests_pytorch/strategies/test_ddp_spawn_strategy.py

```diff
@@ -20,7 +20,7 @@
 
 from lightning.pytorch import LightningModule, Trainer
```

Comment:
I will rename/merge and deduplicate the test files in a follow up. To simplify the work of the reviewer